src/reducers/searchReducer.js

```diff
@@ -0,0 +1,26 @@
+import * as types from '../actions/actionTypes';
+
+const initialState = {
+  search: '',
+  results: [],
+  noResults: false,
+};
+
+export default function searchReducer(state = initialState, action) {
+  switch (action.type) {
+    case types.SEARCH_SUBMIT:
+      return {
+        ...state, results: null, keyWords: action.searchKeyWords, loading: true,
```

Comment:
why is results given the value null?

Reply:
That clears the current search page when the user submits a new search.